src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import cn from 'classnames';
+import { getMovie } from '../../api';
+import { ResponseError } from '../../types/ReponseError';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onAddMovie: (movie: Movie) => void
+}
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
extra typing for explicit values
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```